Hw1.2 branch1

hope2183 opened the pull request.

hope2183 committed:
BranchCommit 1

hope2183 committed:
BranchCommit 1

hope2183 committed:
BranchCommit 1

Avenirre commented on src/ru/skypro/Main.java:
```diff
@@ -3,6 +3,81 @@
+        long d = -11_111_111_111L;
+        float e = 3.1111113f;
+        double f = 3.345678901;
+        System.out.println (a);
```
Нужно добавить char и boolean

Avenirre commented on src/ru/skypro/Main.java:
```diff
@@ -3,6 +3,81 @@
+        System.out.println (f);
+
+    //TASK2//
+        float Boxer1Weight = 78.2f;
```
Имена переменных всегда начинаются с маленькой буквы, это относится и ко всем последующим заданиям

Avenirre commented on src/ru/skypro/Main.java:
```diff
@@ -3,6 +3,81 @@
+        int BananaWeight = 80;
+        int BananaQty = 5;
+        int MilkWeight = 105;
+        int MilkWeightML = 200;
```
milkVolumeMl

Avenirre commented on src/ru/skypro/Main.java:
```diff
@@ -3,6 +3,81 @@
+        int ConversionToKg = 1000;
+        int BananaWeight = 80;
+        int BananaQty = 5;
+        int MilkWeight = 105;
```
milk100mlWeight

Avenirre commented on src/ru/skypro/Main.java:
```diff
@@ -3,6 +3,81 @@
+        int BananaQty = 5;
+        int MilkWeight = 105;
+        int MilkWeightML = 200;
+        int MilkQty = MilkWeightML/100;
```
milkPortions

Avenirre commented on src/ru/skypro/Main.java:
```diff
@@ -3,6 +3,81 @@
+
+    //TASK4//
+
+        byte WeightToLose = 7;
```
weightToLoseKg

Avenirre commented on src/ru/skypro/Main.java:
```diff
@@ -3,6 +3,81 @@
+    //TASK4//
+
+        byte WeightToLose = 7;
+        int DailyLoss1 = 250;
```
dailyLossGr1

Avenirre commented on src/ru/skypro/Main.java:
```diff
@@ -3,6 +3,81 @@
+        byte WeightToLose = 7;
+        int DailyLoss1 = 250;
+        int DailyLoss2 = 500;
+        int DaysForLoss1 = WeightToLose*ConversionToKg/DailyLoss1;
```
Проще один раз перевести 7 кг в граммы и дальше переиспользовать

hope2183 committed:
BranchCommit 1
изменены имена переменных (начинаются с маленькой буквы)

hope2183 committed:
BranchCommit 1
изменены имена переменных (начинаются с маленькой буквы)

hope2183 committed:
BranchCommit 1
изменены имена переменных (начинаются с маленькой буквы)
добавлены переменные логического и символьного типов
